be/src/storage/rowset/column_writer.cpp

```diff
@@ -393,11 +393,6 @@ Status ScalarColumnWriter::init() {
         if (_opts.zone_map_truncate_string) {
             _zone_map_index_builder->enable_truncate_string();
         }
-        if (is_string_type(_type_info->type())) {
-            _zone_map_index_quality_judger =
-                    ZoneMapIndexQualityJudger::create(_type_info.get(), config::string_zonemap_overlap_threshold,
```

Comment:
remove `ZoneMapIndexQualityJudger` related code?

Reply:
It’s still worthwhile; I plan to reintroduce it someday when I have a clearer idea.